song-server/src/main/java/bio/overture/song/server/oauth/ExpiringOauth2Authentication.java

```diff
@@ -7,15 +7,15 @@
 
 @Getter
 public class ExpiringOauth2Authentication extends OAuth2Authentication {
-  private final int expiry; // expiry time of the authentication token in seconds
+  private final long expiry; // expiry time of the authentication token in seconds
```

Comment:
:+1:

Reply:
Just a note, the `AccessTokenConverterWithExpiry` is still using int for the `expiryInSeconds`